### ISSUE TITLE
Greaseweazle missing features

### ISSUE THREAD

**Issue by keirf** [davidgiven/fluxengine#545]
Fluxengine's Greaseweazle documentation states that most features are available when using Greaseweazle hardware. Is there anything lacking in Greaseweazle firmware that would unblock missing features? Or would effort be better directed at fluxengine's Greaseweazle driver?

And a minor documentation nit: Greaseweazle, not GreaseWeazle.

**Comment by davidgiven** [comment]
AFAIK the only major missing feature is hard sector support. I don't have any hard sectored disks and so am unable to test this; FluxEngine's own firmware for this was contributed (replacing my own broken attempt to implement it!).

Re 'GreaseWeazle': muscle memory, I'm afraid. I thought I'd fixed most of these...

**Comment by keirf** [comment]
Yes I agree sync'ing with the sector marks on writes would be great. Oh well. There's so little HS media out there.

**Comment by tdaede** [comment]
Not exactly a feature but more a bug, I still experience https://github.com/davidgiven/fluxengine/issues/512 frequently, I think fluxengine is missing an initialization or resync on connect, but I haven't gone and looked at the details of it yet myself.
